source/controllers/dependencies-between-controllers.md

```diff
@@ -42,14 +42,14 @@ its parent `PostController`, which can be done via `controllers.post`
 </ul>
 ```
```

Comment:
The section above here still has "<h1>Comments for {{controllers.post.title}}</h1>" in it. I think that needs to be changed?